src/FixedMCTS.py

```diff
@@ -2,10 +2,20 @@
 import numpy as np
 
 class FixedMCTS(MCTS):
-    """An implementation of Monte Carlo Tree Search that only aggregates statistics up to a fixed depth."""
-    def __init__(self, maxDepth, explorationRate, threads = 1, timeLimit = None, playLimit = None, **kwargs):
-        self.MaxDepth = maxDepth
-        return super().__init__(explorationRate, timeLimit, playLimit, threads, **kwargs)
+    """ An implementation of Monte Carlo Tree Search that only aggregates 
+        statistics up to a fixed depth.
+    """
+    def __init__(self, parameters):
+        self.parameters = parameters['mcts']
+        self.MaxDepth = self.parameters['maxDepth']
+        explorationRate = self.parameters['explorationRate']
+        timeLimit = self.parameters['timeLimit']
+        playLimit = self.parameters['playLimit']
+        threads = self.parameters['threads']
+
```

Comment:
Can you use parameters.get('key') for these? It makes it easier to test things ad-hoc, since I don't have to fill out every key.

Reply:
<a href="https://github.com/jordan-singer/BlackBird/commit/eb7e34e105f0184c21b526ee51868ac24f1c2a80">Resolved</a>.